files/galaxy/dynamic_rules/usegalaxy/tool_destinations.yaml

```diff
@@ -535,7 +535,7 @@ gemini_load: {cores: 10, mem: 40}
 
 ggplot2_heatmap2: {mem: 24}
 glassgo: {mem: 4, runner: condor_docker}
-graphprot_predict_profile: {mem: 8}
+graphprot_train_predict: {mem: 16}
```

Comment:
lass uns mal die ID gleich behalten ...

Reply:
Ja wo kann ich hier ändern kein Plan ...

Reply:
meistens is es ein Pencil button ;)

Reply:
haha ok habs